js_modules/dagit/packages/core/src/runs/RunsRootNew.tsx

```diff
@@ -208,11 +214,7 @@ export const RunsRoot = () => {
                               ))}
                           </Box>
                         ) : null}
-                        <RunsFilterInput
-                          tokens={mutableTokens}
-                          onChange={setFilterTokensWithStatus}
-                          enabledFilters={enabledFilters}
-                        />
+                        {filtersSlot}
```

Comment:
@bengotow  I resolved the moving the component issue inside `RunTable` but I think the `usePortalSlot` should stay in the root since the `renderError` path moves the component too.